README.md

```diff
@@ -291,14 +292,21 @@ systemd::service_limits { 'foo.service':
 }
 ```
 
-Or provide the configuration file yourself. Systemd reloading and restarting of the service are handled by the module.
+The replacement is to use the `systemd::manage_dropin` defined type directly.
+To migrate from the above example, use the following:
 
 ```puppet
-systemd::service_limits { 'foo.service':
-  source => "puppet:///modules/${module_name}/foo.conf",
+systemd::manage_dropin { 'foo.service-90-limits.conf':
+  unit     => 'foo.service',
+  filename => '90-limits.conf',
+  limits   => {
```

Comment:
`limits => {` is wrong here and causes the following error:
`Systemd::Manage_dropin[foo.service-90-limits.conf]: has no parameter named 'limits' `

Correct syntax is
`service_entry => {`

Reply:
Made #476 to fix this.

Reply:
I wish we had ci on examples ...